test/test_quantized.py

```diff
@@ -82,9 +79,10 @@ def _pool_output_shape(self, input_size, kernel_size, padding, stride,
         return output_size
 
     """Tests the correctness of the quantized::relu op."""
-    @given(Q=qtensor(shapes=array_shapes(1, 5, 1, 5)))
-    def test_qrelu(self, Q):
-        X, (scale, zero_point), (qmin, qmax), torch_type = Q
+    @hy.given(X=hyut.tensor(shapes=hyut.array_shapes(1, 5, 1, 5)),
```

Comment:
btw, how can we pass in float_min, float_max arguments that depend on qparams?

Reply:
That part is very hard. I can think of only two ways -- (1) the way it was implemented before; (2) use `assume` inside of the test to modify the generated tensors

Reply:
can we keep the original API and draw qparams inside the tensor and tensor_conv2d?